tensilelite/Tensile/AsmAddressCalculation.py

```diff
@@ -634,24 +646,44 @@ def incrementToNextRow(self, kernel, tc, ss, stmp, bpeType=None):
                                 shiftHex=log2(tmpBpe), \
                                 comment="incToNextRow: Scale by BPE"))
 
-                if numRows >= 0:
-                    module.add(SAddU32(dst=sgpr("Srd%s+0"%(tc)), \
-                                        src0=sgpr("Srd%s+0"%(tc)), \
-                                        src1=sgpr(stmp), \
-                                        comment="incToNextRow: gra SRD += inc(lower)" ))
-                    module.add(SAddCU32(dst=sgpr("Srd%s+1"%(tc)), \
-                                        src0=sgpr("Srd%s+1"%(tc)), \
-                                        src1=0, \
-                                        comment="incToNextRow: gra SRD += inc(upper)" ))
-                else: # numRows < 0
-                    module.add(SSubU32(dst=sgpr("Srd%s+0"%(tc)), \
-                                        src0=sgpr("Srd%s+0"%(tc)), \
-                                        src1=sgpr(stmp), \
-                                        comment="incToNextRow: gra SRD -= inc(lower)" ))
-                    module.add(SSubBU32(dst=sgpr("Srd%s+1"%(tc)), \
-                                        src0=sgpr("Srd%s+1"%(tc)), \
-                                        src1=0, \
-                                        comment="incToNextRow: gra SRD -= inc(upper)" ))
+                if dst!= -1:
+                    if numRows >= 0:
```

Comment:
Better not to produce redundant codes
if dst == -1:
   dst== "Srd%s"%(tc)
...
if numRows>=0

Reply:
done